.gitignore

```diff
@@ -63,3 +63,5 @@ typings/
 dist/
 
 .idea
+
+package-lock.json
```

Comment:
Undo this and add `package-lock.json` back; You can make a different PR for this but let's not mix things.

Reply:
Sure @ronthecookie